### DISCUSSION TITLE
Is it possible to add a footer to the presentation?

### DISCUSSION THREAD

**Discussion by ghilainm** [hakimel/reveal.js#3110] [Q&A] [answered]
Hellow,

is it possible to add a footer in the presentation with information like:

- Date
- Author

Would be nice if it could be done automatically.

Thanks a lot!

**Comment by Sarah-Thomsen** [comment, marked as answer]
There actually is a plugin called title-footer that creates a footer with just the title of the presentation (https://github.com/e-gor/Reveal.js-Title-Footer). You can customize it by adding parameters (just look at the explaination on the plugins side). There you can exchange the title for any other text you'd like. That's the only way I know. It doesn't work automatically but you can type in the date and author.
If you have any trouble using the plugin I'd be happy to help again.

Hope that helps!

**Comment by peter-lyons-kehl** [comment]
See also #806.
